trunk/src/lsc/myloopdef.py

```diff
@@ -71,6 +75,13 @@ def run_getmag(imglist, _output='', _interactive=False, _show=False, _bin=1e-10,
                 if tel[-1] not in setup:  setup[tel[-1]] = {}
                 if filt[-1] not in setup[tel[-1]]:  setup[tel[-1]][filt[-1]] = {}
 
+                # These three things should be the same for all the images in imglist
+                ftype = ggg[0]['filetype']
+                dtype = ggg[0]['difftype']
+                # Get name of target:
+                namequery = lsc.mysqldef.getfromdataraw(conn, 'targetnames', 'targetid', str(ggg[0]['targetid']))
+                targetname = namequery[0]['name']
```

Comment:
A given object may have more than one name in targetnames. Are you sure you want to just keep the first one?

Reply:
Yes, on the SNEx2 side it'll look up the target with that name, and all of the names for a given target are in SNEx2 now